### ISSUE TITLE
Feature Request: Expose client package version

### ISSUE THREAD

**Issue by blackchineykh** [open-wa/wa-automate-nodejs#586]
Hi, I want to have my nodejs app be able to read the version of the package of the @open-wa/wa-automate the client is using. I noticed that this is read within the client but not exposed through the library. I know there's ways to read this from the file system, but since this is already being read into the client, can this be exposed for use by my nodejs app?

My use case, i want to track both the whatsapp version and wa-automate versions being used

**Comment by smashah** [comment]
Yeh I was planning on doing that tonight anyways :D

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i`);
exec(`npm run release-ci`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [1.9.85](https://github.com/open-wa/wa-automate-nodejs/compare/1.9.84...1.9.85) (2020-06-27)

- build(deps): bump puppeteer from 4.0.0 to 4.0.1 [`#572`](https://github.com/open-wa/wa-automate-nodejs/pull/572)
- build(deps-dev): bump @types/node from 14.0.13 to 14.0.14 [`#573`](https://github.com/open-wa/wa-automate-nodejs/pull/573)
- build(deps-dev): bump @types/line-reader from 0.0.29 to 0.0.30 [`#577`](https://github.com/open-wa/wa-automate-nodejs/pull/577)
- Release 1.9.85 [`6bfb953`](https://github.com/open-wa/wa-automate-nodejs/commit/6bfb9539f099bc19d405f5db05b7695fbfb02c52)
- deprecated `getAllChatsWithMessages` #576 [`f454cd9`](https://github.com/open-wa/wa-automate-nodejs/commit/f454cd9e62bb3229b15308755e9cb74f7bd68129)
- animated stickers (Still WIP), prelim groundwork #580 [`55232ee`](https://github.com/open-wa/wa-automate-nodejs/commit/55232ee9979e99815202af2e53840a0ec2b1e514)
- Feature: onAnyMessage now includes more/all message types #585 [`14b49cc`](https://github.com/open-wa/wa-automate-nodejs/commit/14b49ccf3d7249fd6168e145a2294b5e039f40b1)
- Feature: implemented `iAmAdmin` #584 [`fb1354e`](https://github.com/open-wa/wa-automate-nodejs/commit/fb1354ee60bd9054b45aa9ec5872dd4b3d28b4dc)
- Feature: Implemented `getSessionInfo` #586 [`fef6b95`](https://github.com/open-wa/wa-automate-nodejs/commit/fef6b95c095c5a4ceeda3ffe2ae1bc94d58e76b8)
- Improved error messages #581 [`03dcde7`](https://github.com/open-wa/wa-automate-nodejs/commit/03dcde71d55cfa41a854da2e4afce2433d23d39e)
- chore: bump supported WA version [`3d77906`](https://github.com/open-wa/wa-automate-nodejs/commit/3d77906ca30f2e9aa86a9f5b5287e3a34ab401e6)
- Demo: increases express json limit to enable sending base64 over api [`83bfbf7`](https://github.com/open-wa/wa-automate-nodejs/commit/83bfbf7ebc58198b2c89118b2a34b39360040d0d)
- Enhancement: Improved performance 5 second faster load times if safeMode is off. [`e3cefef`](https://github.com/open-wa/wa-automate-nodejs/commit/e3cefefbd3af3848938f73a5c03a5df32fe8b15a)
- removed p-queue as dev dep (already a standard dep) [`7622cce`](https://github.com/open-wa/wa-automate-nodejs/commit/7622cceba23eae14f101c138c80cf0b61779cef0)

**Comment by smashah** [comment]
Implemented in 1.9.85

```javascript
const sessionInfo = client.getSessionInfo();
// returns the debug info from initilization

const config = client.getConfig(); //available after 1.9.86
// returns the config used to create the session without any sensitive data

```

https://open-wa.github.io/wa-automate-nodejs/classes/client.html#getsessioninfo
https://open-wa.github.io/wa-automate-nodejs/classes/client.html#getconfig
